src/python/pants/core/goals/lint_test.py

```diff
@@ -128,13 +127,13 @@ def make_target_with_origin(address: Optional[Address] = None) -> TargetWithOrig
     @staticmethod
     def run_lint_rule(
         *,
-        field_set_collection_types: List[Type[LinterFieldSets]],
+        field_set_collection_types: List[Type[LintRequest]],
```

Comment:
Bad parameter name. There will likely be a couple other places to change, too. Please grep `field_sets`.

(I've renamed this type a lot 😂)